numpy/typing/tests/data/fail/lib_utils.pyi

```diff
@@ -1,5 +1,3 @@
 import numpy as np
 
-np.byte_bounds(1)  # E: incompatible type
```

Comment:
Shouldn't these typing tests stay, just under the new name?

Reply:
Sure! Now I import it from the new place.